addons/account/models/account_invoice.py

```diff
@@ -45,6 +45,8 @@ class AccountInvoice(models.Model):
     _description = "Invoice"
     _order = "date_invoice desc, number desc, id desc"
 
+    SAMPLE_INVOICE_NAME = 'SAMPLE_INVOICE'
```

Comment:
Based on name ? What about a field, is a constant is used for name what about account and journal

Reply:
A variable seemed lighter, it is used only to prevent validating the invoice and that feature is not needed (I think) for account or journal.